test/e2e/set/scyllacluster/scyllacluster_pv.go

```diff
@@ -82,17 +83,18 @@ var _ = g.Describe("ScyllaCluster Orphaned PV controller", func() {
 		sc.Spec.AutomaticOrphanedNodeCleanup = true
 		sc.Spec.Datacenter.Racks[0].Members = 3
 		sc.Spec.Datacenter.Racks[0].Storage.StorageClassName = pointer.Ptr(testStorageClassName)
-		sc.Spec.Datacenter.Racks[0].Placement = &scyllav1.PlacementSpec{
-			PodAffinity: &corev1.PodAffinity{
-				RequiredDuringSchedulingIgnoredDuringExecution: []corev1.PodAffinityTerm{
-					{
-						LabelSelector: &metav1.LabelSelector{
-							MatchLabels: map[string]string{
-								cloneLabelKey: f.Namespace(),
-							},
+		if sc.Spec.Datacenter.Racks[0].Placement == nil {
+			sc.Spec.Datacenter.Racks[0].Placement = &scyllav1.PlacementSpec{}
+		}
+		sc.Spec.Datacenter.Racks[0].Placement.PodAffinity = &corev1.PodAffinity{
+			RequiredDuringSchedulingIgnoredDuringExecution: []corev1.PodAffinityTerm{
```

Comment:
wouldn't it be more plugable to append here?

Reply:
ok